app/konnect/centralized-consumer-management.md

```diff
@@ -43,11 +43,11 @@ You can manage consumers centrally using the {{site.konnect_short_name}} API. On
    Be sure to replace the following with your own values:
    * `{region}`: Region for your {{site.konnect_short_name}} instance. Data planes can only reach out to realms in the same region as the data plane.
    * `$KONNECT_TOKEN`: Replace with your {{site.konnect_short_name}} personal access token.
-   * `$CONTROL_PLANE_UUID`: (Optional) Replace with your control plane UUID.
+   * `$CONTROL_PLANE_UUID`: (Optional) Replace with your control plane UUID. If you don't specify a control plane, this means that no control plane can access the consumers in the realm.
```

Comment:
I wouldn't call the variables `_UUID`, just call the example variable name the `CONTROL_PLANE_ID` and in the instructions I would just say "replace with your control plane ID"